src/components/Decision.js

```diff
@@ -63,20 +63,34 @@ const icon={
 }
 
 export class Decision extends Component {
-
-    
+   constructor(props){
+        super(props)
+        this.state = {
+            ans: props.randAns
+        }
+   }   
+   handleRandomAns = ()=> {
+    let options,randomValue;
+    options = this.props.options
+    randomValue = options[Math.floor(Math.random() * options.length)]        
+       this.setState({
+           ans: randomValue
+       })
+   }
```

Comment:
oh you made this component has its own logic, what's the advantage of that to placing the entire logic in one place
where is 'randAns' coming from, is it a value passed through props to this component?